Src/EB/AMReX_distFcnElement.cpp

```diff
@@ -10,7 +10,7 @@ int distFcnElement2d::solve_thomas(std::vector<amrex::Real> ain,
                                    std::vector<amrex::Real> bin,
                                    std::vector<amrex::Real> cin,
                                    std::vector<amrex::Real> din,
-                                   std::vector<amrex::Real>& x) {
+                                   std::vector<amrex::Real> &x) {
   std::vector<amrex::Real> a, b, c, d;
```

Comment:
`a`, `b`, `c`, and `d` are passed by value.  Thus there are no need to create local copies.  Furthermore, `a` and `c` are not modified, so they can be passed as `const&`.